orangecontrib/educational/widgets/owpolynomialregression.py

```diff
@@ -266,6 +266,8 @@ def apply(self):
             data_table = Table(
                 Domain([attributes], class_vars=[class_var]), self.data)
 
+            mask = ~(np.isnan(data_table.X).any(axis=1) | np.isnan(data_table.Y))
+            data_table = data_table[mask]
```

Comment:
Could this work with `DropInstances` or `IsDefined` imputation preprocessors?

Reply:
`DropInstances´: Yes, but only one column a time.

Yes, could work with `IsDefined´. This can accept more columns at the same time, but selected only as indices not variables.

So I think it is better to do this directly with `numpy`.

Reply:
Well, I think `IsDefined` would work. When `columns` are set to `None` all columns are checked.